example/arcfullsim.py

```diff
@@ -104,22 +104,26 @@ def setupCerenkov(kernel):
         outImagePrefix = "arcsim_"
         ROOT.gROOT.SetBatch(1)
         rootfile = ROOT.TFile(SIM.outputFile)
-        if not "edm4hep" in SIM.outputFile:
+        number_of_events = []
+        if "edm4hep" not in SIM.outputFile:
```

Comment:
There is a bit of duplication in this part. Maybe it could be condensed in a single one and use f-strings to decide when it's EDM4hep output or not? Some cuts also look different, for example, non-edm4hep:
```
            n1 = EVENT.Draw(
                "ArcCollection.position.Z():ArcCollection.position.phi()",
                "((ArcCollection.cellID>>5)&0x7)==0",
            )
```
versus with EDM4hep:
```
            n1 = EVENT.Draw(
                "ArcCollection.position.z:atan(ArcCollection.position.y/ArcCollection.position.x)",
                "((ArcCollection.cellID>>5)&0x7)==0&& ArcCollection.position.x>0",
            )
```
x is required to be positive with EDM4hep but not without?